packages/create-itp-react-app/templates/app/src/core/reducers/index.ts

```diff
@@ -0,0 +1,14 @@
+import { combineReducers } from 'redux';
+import entities from './entities';
+import ui from './ui';
+
+const rootReducer = () => {
+  const appReducer = combineReducers({
+    entities,
+    ui,
+  });
+
+  return (state: any, action: any) => appReducer(state, action);
```

Comment:
is there no type in redux that can be imported i/o any here?

Reply:
looking into https://github.com/piotrwitek/typesafe-actions

Reply:
using [typesafe-actions](https://github.com/piotrwitek/typesafe-actions) now for the redux flow